rest-service/manager_rest/rest/resources_v3/tenants.py

```diff
@@ -177,6 +177,34 @@ def put(self, multi_tenancy):
         )
 
     @rest_decorators.exceptions_handled
+    @authorize('tenant_update_group', get_tenant_from='data')
+    @rest_decorators.marshal_with(TenantResponse)
+    @rest_decorators.no_external_authenticator('update group in tenant')
+    def patch(self, multi_tenancy):
+        """Update role in group tenant association."""
+        request_dict = rest_utils.get_json_and_verify_params(
+            {
+                'tenant_name': {
+                    'type': unicode,
+                },
+                'group_name': {
+                    'type': unicode,
+                },
+                'role': {
+                    'type': unicode,
+                },
+            },
+        )
+        rest_utils.validate_inputs(request_dict)
+        role_name = request_dict['role']
+        rest_utils.verify_role(role_name)
```

Comment:
the default of verify_role is is_system_role=False
and we allow both system_wide and tenant_wide roles, so need to update the function..

Reply:
Why do we allow both system and tenant roles to update the role in the group tenants relationship? My understanding is that system roles are only allowed in the `users_roles` table and that for relationship tables (`users_tenants` and `groups_tenants`) only tenant roles are allowed. Is this correct?

Also, note that setting `is_system_role=True` here would be inconsistent with the `put` method a few lines above in which `is_sistem_role=False`.